docs/man/policy/provider-config/get.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Get a Provider Config
+command:
+  name: get
+  aliases:
+    - g
+  flags:
+    - name: id
+      shorthand: i
+      description: ID of the provider config to retrieve
+      required: false
+    - name: name
+      shorthand: n
+      description: Name of the provider config to retrieve
+      required: false
```

Comment:
Optional nit: you can skip `required: false` in these docs as it's the default state.